docs/source/modules/mixedlayer.rst

```diff
@@ -0,0 +1,277 @@
+are then used to compute the Q-flux by running the offline python script: src/extra/python/scripts/calculate_qflux/calculate_qflux.py. This script produces a .nc file which is then passed into the model 
+(qflux_file_name).
+
+See Q-flux options below for namelist options. Note that the q-flux is only relevant for slab ocean experiments (not fixed or prescribed SST runs). Also note that if the MiMA radiation code is used then the 
+Q-flux is implemented following Merlis et al 2013 [MerlisEtAl2013]_
+
+More information on the method for Q-flux can be found in Russel et al 1985 [RusselEtAl1985]_
+
+
+Namelist options
+----------------
+
++-------------------+------------------------------------------------------------+---------+
+| Option            | Summary                                                    |Default  |
++===================+============================================================+=========+
+|``evaporation``    |Switch for surface evaporation.                             |``True`` |
++-------------------+------------------------------------------------------------+---------+
+|``depth``          |Mixed layer depth.                                          | ``40.0``|
++-------------------+------------------------------------------------------------+---------+
+
+Q-flux options
+^^^^^^^^^^^^^^^^^^^^
+If ``do_qflux`` is True, use ``qflux_amp`` and ``qflux_width`` to calculate a time-independent surface q-flux.
+If ``load_qflux`` is True, use input file to load in a time-independent or time-dependent q-flux.
+
++-------------------+----------------------------------------------------------------+---------+
+| Option            | Summary                                                        |Default  |
++===================+================================================================+=========+
++-------------------+----------------------------------------------------------------+---------+
+|``do_qflux``       | Switch to calculate time-independent q-flux.                   |``False``|
++-------------------+----------------------------------------------------------------+---------+
+|``qflux_amp``      | Amplitude of time-independent q-flux if ``do_qflux`` is True.  | ``0.0`` |
++-------------------+----------------------------------------------------------------+---------+
+|``qflux_width``    | Width of time-independent q-flux if ``do_qflux`` is True.      | ``16.0``|
++-------------------+----------------------------------------------------------------+---------+
+
++----------------------+----------------------------------------------------------------------------------------------------------------------------------------------------------------------------+-----------------+
+| Option               | Summary                                                                                                                                                                    |Default          |
```

Comment:
Why are these as 2 tables? I think line wrapping would look better than a scrolling bar. What is your perspective?